arangod/RestServer/QueryRegistryFeature.cpp

```diff
@@ -148,14 +148,14 @@ void QueryRegistryFeature::prepare() {
 
   // create the query registery
   _queryRegistry.reset(new aql::QueryRegistry(_queryRegistryTTL));
-  QUERY_REGISTRY = _queryRegistry.get();
+  QUERY_REGISTRY.store(_queryRegistry.get(), std::memory_order_relaxed);
 }
 
 void QueryRegistryFeature::start() {}
 
 void QueryRegistryFeature::unprepare() {
   // clear the query registery
-  QUERY_REGISTRY = nullptr;
+  QUERY_REGISTRY.store(nullptr, std::memory_order_relaxed);
```

Comment:
No, here a relaxed-store is sufficient. The other store has to use `memory_order_release` to order operations on the _members_ of the pointer that gets stored, i.e., we want to avoid data races on the registry members.